excel-importer/src/com/axonivy/util/excel/importer/wizard/ExcelImportProcessor.java

```diff
@@ -139,13 +140,18 @@ private void importExcel(IProjectDataClassManager manager, FileResource excel, I
     );
   }
 
-  private List<?> importData(Sheet sheet, IEntityClass newEntity, IProcessModelVersion pmv) throws SQLException {
+  private List<?> importData(Sheet sheet, IEntityClass newEntity, IProcessModelVersion pmv) throws Exception {
     var persist = pmv.getAdapter(IPersistenceContext.class);
     var ivyEntities = persist.get(selectedPersistence);
     EntityDataLoader loader = new EntityDataLoader(ivyEntities);
-    var entityType = loader.createTable(newEntity);
-    loader.load(sheet, newEntity);
-    List<?> loaded = ivyEntities.findAll(entityType);
+
+    var system = pmv.getApplication().getSecurityContext().sessions().systemUser();
+    var request =  RequestFactory.createRestRequest(pmv, system);
+    List<?> loaded = new RequestContext(request).callInContext(() -> {
```

Comment:
this provides the 'IRequest' context, which is usually provided via Ivy Tomcat servlet